code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -75,24 +77,19 @@
 
 	location.active_hotspot = src
 
-	if(volume > CELL_VOLUME*0.95)
-		bypassing = TRUE
-	else
-		bypassing = FALSE
+	bypassing = !just_spawned && (volume > CELL_VOLUME*0.95)
```

Comment:
This variable has [other possible effects](https://github.com/tgstation/tgstation/blob/master/code/modules/atmospherics/environmental/LINDA_fire.dm#L189). Is this still valid?

Reply:
Note that the other effects are in process(), which returns early if just_spawned is true. As such, this is equivalent to the old method except that it won't just do nothing on the first bypass.